apps/site/lib/site_web/templates/event/_event_list.html.eex

```diff
@@ -1,13 +1,21 @@
 <div class="page-section event-listing">
-  <%= content_tag(:h2, @title, extra_classes) %>
   <!-- This list-group stuff is bootstrap -->
   <ul class="list-group list-group-flush">
+    <!-- Previous months button. A JS script sets up a listener for button click -->
+    <li class="list-group-item m-previous-events-button">
+      <a class="m-previous-events-button">
+        View Previous <%= pretty_date(Timex.now("America/New_York"), "{Mfull} {YYYY}") %> Events
+        <i class="fa fa-angle-down down" aria-hidden="true"></i>
+      </a>
+    </li>
     <%= for event_teaser <- @events do %>
       <%
         range = %{start: event_teaser.date, stop: event_teaser.date_end}
         path_fn = fn -> cms_static_page_path(@conn, event_teaser.path) end
       %>
-      <li class="list-group-item u-flex-container m-event">
+      <!-- If date is upcoming, display on init -->
```

Comment:
I wonder if this could be simplified to just show if the event hasn't yet ended (so, not having to use the whole range). This would also not hide it if the event's already started but if still ongoing.

Reply:
Ah I wish, but not all events have a stop datetime - so if there is a stop time, that's used.  Otherwise we have to use the start time